.github/workflows/kind-integration-byoargo.yml

```
@@ -0,0 +1,65 @@
+  cancel-in-progress: true
+
+env:
+  GIT_WORKSPACE: ${{ github.workspace }}
+
+jobs:
+  dspo-byoargo-tests:
+    runs-on: ubuntu-latest
+    timeout-minutes: 60
+
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)